server/plugin/kv_test.go

```diff
@@ -74,11 +76,112 @@ func TestStoreUser(t *testing.T) {
 	} {
 		t.Run(test.description, func(t *testing.T) {
 			defer monkey.UnpatchAll()
-			s := pluginStore{}
+			ps := pluginStore{}
 			test.setupTest()
 
-			err := s.StoreUser(&serializer.User{})
+			err := ps.StoreUser(&serializer.User{})
 			assert.EqualValues(t, test.expectedError, err)
 		})
 	}
 }
+
+func TestDeleteUser(t *testing.T) {
+	defer monkey.UnpatchAll()
+	ps := new(pluginStore)
+	p := Plugin{}
+	ps.userKV = kvstore.NewHashedKeyStore(kvstore.NewPluginStore(p.API), UserKeyPrefix)
+	for _, test := range []struct {
+		description   string
+		setupTest     func()
+		expectedError error
+	}{
+		{
+			description: "User is not loaded from the KV store using mattermostUserID",
+			setupTest: func() {
+				monkey.PatchInstanceMethod(reflect.TypeOf(ps), "LoadUser", func(*pluginStore, string) (*serializer.User, error) {
+					return nil, fmt.Errorf("mockError")
```

Comment:
```suggestion
					return nil, fmt.Errorf("mockError")
```
Use something meaningful instead of "mockError". check everywhere

Reply:
We have used this at many places and in other plugins as well. I think it's fine to use it

Reply:
It was removed from VA. If it doesn't take much time we can fix it here as well. Otherwise this also fine.

Reply:
should we create a ticket for this, and update this after all test cases are merged @manojmalik20 ?

Reply:
Yes @ayusht2810 You can do this. Please create a ticket.

Reply:
https://brightscout.atlassian.net/browse/MI-2351?atlOrigin=eyJpIjoiMDk1OTBmMDU0NDU3NDlkM2FjZTJkNTRkZjYzMGY2ZTEiLCJwIjoiaiJ9

created ticket for this